api/routes.go

```diff
@@ -2,13 +2,15 @@ package api
 
 import (
 	"github.com/go-macaron/binding"
+	"github.com/go-macaron/gzip"
 	"github.com/raintank/metrictank/api/middleware"
 	"github.com/raintank/metrictank/api/models"
 	"gopkg.in/macaron.v1"
 )
 
 func (s *Server) RegisterRoutes() {
 	r := s.Macaron
+	r.Use(gzip.Gziper())
```

Comment:
i added a config option, as it will be beneficial to disable compression when debugging so that you can look at the request/respons data using ngrep or other network tool